src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentProjects/DefinitionProject.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.FindSymbols.DependentProjects
+{
+    internal readonly struct DefinitionProject : IEquatable<DefinitionProject>
```

Comment:
extracted out.  unchanged otherwise.

Reply:
Is it worth a quick doc comment for what this represents? In isolation it's a bit odd.

Reply:
done.